e2e/test/scenarios/downloads/downloads.cy.spec.js

```diff
@@ -150,6 +150,29 @@ describe("scenarios > question > download", () => {
   });
 });
 
+describe("scenarios > dashboard > download pdf", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+  it("should allow you to download a PDF of a dashboard", () => {
+    cy.createDashboardWithQuestions({
+      dashboardName: "saving pdf dashboard",
+      questions: [canSavePngQuestion, cannotSavePngQuestion],
+    }).then(({ dashboard }) => {
+      visitDashboard(dashboard.id);
+    });
+
+    cy.get("main header").within(() => {
+      cy.icon("ellipsis").click();
```

Comment:
it would be beneficial to add an `aria-label` to the button and query the button using it. Also `main header` relies on the current structure a lot and potentially where can be many headers. I think we can assign a test-id for that

Reply:
Whoops, I had forgotten I hadn't updated that. I'm going to add a test-id for dashboard header actions/buttons

Reply:
For buttons it is better to add `aria-label` because it also makes the app more accessible